content/applications/productivity/calendar.rst

```diff
@@ -9,3 +10,206 @@ Calendar
+- :guilabel:`Organizer`: This is defaulted to the current Odoo user. Select a new Organizer from
+  existing contacts or create and edit a new contact.
+- :guilabel:`Reminders`: Select notification options to sent to attendees. Choose a default
+  notification or configure new reminders.
+
+Coordinate with teams
+---------------------
+
+When scheduling an event for multiple users, tick the checkbox next to :guilabel:`Attendees` to
+view team members' availability. Tick the checkbox next to users to show or hide individual
+calendars.
+
+.. image:: calendar/calendar-attendees.png
+   :align: center
+   :alt: View of Attendees section on Calendar app.
+
+Share Availabilities
+--------------------
+
+Click the :guilabel:`Share Availabilities` box to share external meeting invitations. On the
+calendar, select the target time and date to add them as options in the invitation. Once
+availability has been selected, click the :guilabel:`Open` button to navigate to the associated
+meeting.
+
+.. image:: calendar/calendar-meeting-share-availability.png
+   :align: center
+   :alt: Share availability window on Calendar app.
+
+On the meeting page, users have several configuration options:
+
+Under :guilabel:`Scheduling`, select a minimum hour window to ensure that appointments are
+confirmed a specified amount of time in advance. For example, select :guilabel:`01:00` to require
+attendees to confirm at least one hour before their appointment time.
+
+Under :guilabel:`Allow Cancelling` select a minimum hour window required before the appointment if
+attendees need to make a cancellation.
+
+Select *Users* or *Resources* under the :guilabel:`Availability on` section to allow invitees
+to select internal users to meet or supplies like tables to reserve for their meeting.
+
+The :guilabel:`Availability on` section enables attendees to book *Users* or *Resources* such as
+meeting rooms or tables for their appointment. Select *Users* or *Resources*, then type in the
+contact or resource in the space below.
+
+The :guilabel:`Front-End Display` option is used to show *User* or *Resource* photos on
+the appointment page.
+
+If *Resources* has been selected, users have an option to :guilabel:`Manage Capacities`. Tick the
+checkbox to limit the maximum amount of people that can use the resource at the same time.
+
+The :guilabel:`Assign Method` section enables the order in which attendees book their time and
+User/Resource:
+
+- *Pick User/Resource then Time*
+
+- *Select Time then User/Resource*
+
+- If *Resources* has been selected under the :guilabel:`Availability On` section, there
+  will be a third option: *Select Time then auto-assign*.
+
+Schedule
+~~~~~~~~
+
+In the :guilabel:`Schedule` tab of the new meeting page, add or delete time slots.
+
+.. image:: calendar/calendar-new-meeting.png
+   :align: center
+   :alt: New meeting page on Calendar app.
+
+Options
+~~~~~~~
+
+The :guilabel:`Options` tab provides additional configurations:
```

Comment:
I'm seeing some slightly different options in the 'Options' tab of the 17.0 runbot I'm using for this review:
https://drive.google.com/file/d/192matSbX4gFquxxaj12QH0_9zoTtfhvG/view?usp=sharing

Might be best to double-check these during your revisions to ensure the accuracy. Also, listing them in the order they appear, top-down on the left, then top-down on the right... helps readers follow along. 

Also, you can remove the empty lines between each field explanation 👍 

(adhere this suggestion for lines 160-195)